packer/scripts/guestAdd.sh

```diff
@@ -17,3 +17,5 @@ ${MNT}/VBoxLinuxAdditions.run --nox11 || true
 
 umount ${MNT}
 rm -rf ${MNT} ${BASE}/VBoxGuestAdditions_${VBX_VERSION}.iso
+cat /var/log/vbox*
```

Comment:
this should surely not be there

Reply:
@Fdall says it's useful for understanding if the imagine build really went well